package.json

```diff
@@ -31,6 +31,7 @@
     "@types/debug": "^4.1.7",
     "debug": "^4.1.1",
     "dottie": "^2.0.0",
+    "ibm_db": "^2.8.0",
```

Comment:
Does this need to be in dependencies? Can it not be in peerDependenciesMeta with the other connectors?

Reply:
Done. Moved the same to DevDependencies and in peerDependenciesMeta as optional like other connectors.